src/api/model/boards.js

```diff
@@ -1,21 +1,36 @@
 const pool = require('./pool');
 
+const formatRow = row => {
```

Comment:
Could you test using the sql method of `camel_case as camelCase` and see how well it works, if we can offload this logic to sql would save us having this code. I think camelCase in postgres will need `camel_case as "camelCase"`

Reply:
I've pushed up a commit - let me know what you think. It's unfortunate you have to list out all the column names in SQL this way, but I think it's better to do the renaming higher up so I think this is nicer.

Also fixed a bug with GraphQL `deleteBoard` mutation always erroring

Reply:
Sweet! Yeah you're right having to do `returning x, x, x` is unfortunate but yep i think this is the way to go 👍